modals/src/main/java/com/decathlon/vitamin/compose/modals/VitaminModals.kt

```diff
@@ -222,7 +223,11 @@ internal fun Buttons(
                 .padding(top = topPadding),
             horizontalArrangement = Arrangement.SpaceBetween
         ) {
-            neutralButton?.invoke()
+            if (neutralButton == null) {
+                Box(modifier = Modifier)
+            } else {
+                neutralButton.invoke()
+            }
```

Comment:
I'm not sure to understand this fix 🤔 What is the difference with the preview implementation?